prboom2/src/g_game.c

```diff
@@ -3179,13 +3182,71 @@ void G_BeginRecording (void)
 {
   int i;
   byte *demostart, *demo_p;
+  int num_requirements = 0;
   demostart = demo_p = malloc(1000);
   longtics = 0;
 
-  if (umapinfo_loaded)
+  // ano - jun2019 - add the extension format if needed
+  if (gamemapinfo)
+  {
+    num_requirements++;
+  }
+
+  if (num_requirements > 0)
   {
-	  *demo_p++ = 255;
+    // demover
+    *demo_p++ = 0xFF;
+    // signature
+    *demo_p++ = 'P';
+    *demo_p++ = 'R';
+    *demo_p++ = 'B';
+    *demo_p++ = '+';
```

Comment:
If Eternity is considering also support for these demos (if UMAPINFO support lands on Eternity), it might also be advisable to agree with them on adding another '\0' as separator after the version number, since Eternity uses 4 bytes for the version.

Reply:
If Eternity is considering also support for these demos (if UMAPINFO support lands on Eternity), it might also be advisable to agree with them on adding another '\0' as separator after the version number, since Eternity uses 4 bytes for the version.

Reply:
If Eternity is considering also support for these demos (if UMAPINFO support lands on Eternity), it might also be advisable to agree with them on adding another '\0' as separator after the version number, since Eternity uses 4 bytes for the version.

Reply:
Ahh crap I think that was my fault from way back when, providing faulty info. Also yes I do plan to bring playback support into EE, as well as recording of cl2.